.github/workflows/build.yaml

```diff
@@ -178,10 +177,7 @@ jobs:
       - name: Build arm 64 dist
         run: |
           mkdir pkgcache
-          docker run --rm --platform linux/arm64 -v $(pwd):/app -w /app -e PKG_CACHE_PATH=pkgcache node:16.13 /bin/bash -c "npm set-script prepare \"\" && npm install && npm i -g @babel/cli @babel/preset-env pkg && npm run create-linux-arm64-dist"
-
-      - name: Make executable
```

Comment:
This file comes out executable already and when I switched the filename and tested with node 18, this failed because we don't have permissions to change this file.  Removed because it isn't needed.